cluster/manifests/skipper/configmap-open-policy-agent.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if eq .ConfigItems.skipper_open_policy_agent_enabled "true" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    application: skipper-ingress
+    component: ingress
+  name: open-policy-agent-config
+  namespace: kube-system
+data:
+  opaconfig.yaml: |-
+    discovery:
+      name: discovery
+      resource: v1/bundles/zalando-apps/{{ `{{ .bundlename }}` }}/discovery.tgz
+      service: styra-bundles
+    labels:
+      environment: {{ .Cluster.Environment }}
+      kubernetes-cluster-alias: {{ .Cluster.Alias }}
+      aws-account-id: "{{ accountID .Cluster.InfrastructureAccount }}"
+    services:
+    - credentials:
+        bearer:
+          token: "{{ `{{ .Env.STYRA_TOKEN }}` }}"
+      name: styra
+      url: "{{ .ConfigItems.skipper_open_policy_agent_observability_url }}"
+    - credentials:
+        s3_signing:
+          web_identity_credentials:
+            aws_region: eu-central-1
```

Comment:
Let's use `{{ .Cluster.Region }}` here.

Reply:
I'm assuming that `eu-west-1` would be the appropriate value if the cluster was in the `eu-west-1` region. If that's not the case disregard my comment.

Reply:
We decided to skip this for now because it's not clear whether this has to be the same region as the cluster.